exotic/plots.py

```diff
@@ -140,19 +140,19 @@ def plot_flux(times, targ, targ_unc, ref, ref_unc, norm_flux, norm_unc, airmass,
 def plot_variable_residuals(save):
     plt.title("Stellar Variability Residuals")
     plt.ylabel("Residuals (flux)")
-    plt.xlabel("Time (BJD_TDB)")
+    plt.xlabel("Time (JD)")
     plt.legend()
     plt.savefig(Path(save) / "temp" / f"Variable_Residuals.png")
     plt.close()
 
 
-def plot_stellar_variability(vsp_params, save, s_name, label):
+def plot_stellar_variability(vsp_params, save, s_name, vsp_auid_comp):
     for vsp_p in vsp_params:
         plt.errorbar(vsp_p['time'], vsp_p['mag'], yerr=vsp_p['mag_err'], color="tomato", fmt='.')
 
-    plt.title(f"{s_name} (Label: {label})")
+    plt.title(f"{s_name} (Label: {vsp_auid_comp})")
     plt.ylabel("Vmag")
-    plt.xlabel("Time (BJD_TDB)")
+    plt.xlabel("Time (JD)")
```

Comment:
Isn't this still in BJD_TDB? We do the conversions ourselves to BJD_TDB, so I think this label is still accurate.

Reply:
AAVSO requires us to save the it in the JD time format. I saved the corresponding JD times to the BJD ones. For reference, here is [AAVSO's](https://www.aavso.org/aavso-extended-file-format) available time formats:

`
- **DATE**: The format of the date used in the report. Times are midpoint of the observation. Convert all times from 
- UT to one of the following formats:
- JD: Julian Day (Ex: 2454101.7563)
- HJD: Heliocentric Julian Day
- EXCEL: the format created by Excel's NOW() function (Ex: 12/31/2007 12:59:59 a.m )
`

Reply:
Hm. Weird. I would think that BJD_TDB is still ok if it is asking for HD.

Can you please ping some folks on our Slack (or let me know if you'd prefer me to take care of this)? Looks like Chuck C and Daniel Z would know the answer to this question, and if not, Bert Pablo should.

Reply:
Bert replied the following when asked if we can use BJD_TDB:
> it is not. We accept HJD and JD but we don’t have a way to accept anything else currently